examples/cifar/train_cifar_custom_loop.py

```diff
@@ -76,6 +77,17 @@ def main():
     optimizer.setup(model)
     optimizer.add_hook(chainer.optimizer.WeightDecay(5e-4))
 
+    if args.resume != '':
```

Comment:
Which do you prefer?
1. `if args.resume != ''`
2. `if args.resume`

Reply:
The former, since it's explicit about the type, so you can keep the current code.